prepare_ui_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+### Download Docker Images
+echo "you must have permission to run docker."
+docker pull selenoid/vnc_chrome:80.0
+docker pull selenoid/video-recorder:latest-release
+docker pull aerokube/selenoid-ui
+
+###create env file
+echo PWD=${PWD} > .env
+
+###
+docker-compose up -d
+sleep 20
```

Comment:
May not be enough on slower machines. Could we use a health check maybe?
Pls see if you can adapt this (credit @AlekSi):  `docker run --health-cmd='curl -f http://127.0.0.1/ping || exit 1' --health-interval=3s --health-interval=2s --health-start-period=10s --health-retries=3`

Reply:
It can be added directly to docker-compose.yml: https://docs.docker.com/compose/compose-file/#healthcheck

Reply:
But then there should be `until healthy` loop anyway.

Reply:
it is a good suggestion but not really wanting to do it right in this revision, thank you.

Reply:
Ok, will add later.